vmdb/app/models/orchestration_stack_openstack_infra.rb

```diff
@@ -0,0 +1,2 @@
+class OrchestrationStackOpenstackInfra < OrchestrationStack
+end
```

Comment:
This was the first place I needed it in https://github.com/Ladas/manageiq/commit/cf71ec3dc60153913af8cd560616b9c7eae56524